--- origsrc/GeoIPJava-1.2.3/BenchmarkGeoIP.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/BenchmarkGeoIP.java	2009-12-01 02:01:44.966215400 -0600
@@ -5,7 +5,7 @@ class BenchmarkGeoIP {
 
 static void testGeoipCountry(int flags,String msg,int numlookups){
   try {
-  LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIP.dat",flags);
+  LookupService cl = new LookupService("/usr/share/GeoIP/GeoIP.dat",flags);
   int i4 = 0;
   long t1 = System.currentTimeMillis();
   for (int i2 = 0;i2 < numlookups;i2++){
@@ -23,7 +23,7 @@ static void testGeoipCountry(int flags,S
 
 static void testGeoipRegion(int flags,String msg,int numlookups){
   try {
-  LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIPRegion.dat",flags);
+  LookupService cl = new LookupService("/usr/share/GeoIP/GeoIPRegion.dat",flags);
   int i4 = 0;
   long t1 = System.currentTimeMillis();
   for (int i2 = 0;i2 < numlookups;i2++){
@@ -41,7 +41,7 @@ static void testGeoipRegion(int flags,St
 
 static void testGeoipCity(int flags,String msg,int numlookups){
   try {
-  LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIPCity.dat",flags);
+  LookupService cl = new LookupService("/usr/share/GeoIP/GeoIPCity.dat",flags);
   int i4 = 0;
   long t1 = System.currentTimeMillis();
   for (int i2 = 0;i2 < numlookups;i2++){
--- origsrc/GeoIPJava-1.2.3/CityLookupTest.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/CityLookupTest.java	2009-12-01 02:01:49.661823600 -0600
@@ -9,7 +9,7 @@ import java.io.IOException;
 class CityLookupTest {
     public static void main(String[] args) {
 	try {
-	    LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIPCity.dat",
+	    LookupService cl = new LookupService("/usr/share/GeoIP/GeoIPCity.dat",
 					LookupService.GEOIP_MEMORY_CACHE );
             Location l1 = cl.getLocation("213.52.50.8");
             Location l2 = cl.getLocation(args[0]);
--- origsrc/GeoIPJava-1.2.3/CountryLookupTest.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/CountryLookupTest.java	2009-12-01 02:01:56.588235700 -0600
@@ -15,7 +15,7 @@ class CountryLookupTest {
 	    // String dir = System.getProperty("user.dir"); 
 
 	    // Uncomment for Linux
-	    String dir = "/usr/local/share/GeoIP";
+	    String dir = "/usr/share/GeoIP";
 
 	    String dbfile = dir + sep + "GeoIP.dat"; 
 	    // You should only call LookupService once, especially if you use
--- origsrc/GeoIPJava-1.2.3/NetspeedLookupTest.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/NetspeedLookupTest.java	2009-12-01 02:02:10.503460000 -0600
@@ -9,7 +9,7 @@ import java.io.IOException;
 class NetspeedLookupTest {
     public static void main(String[] args) {
         try {
-            LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIPNetspeed.dat");
+            LookupService cl = new LookupService("/usr/share/GeoIP/GeoIPNetspeed.dat");
             if (args.length > 0) {
                 int speed = cl.getID(args[0]);
 	        if (speed == cl.GEOIP_UNKNOWN_SPEED){
--- origsrc/GeoIPJava-1.2.3/OrgLookupTest.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/OrgLookupTest.java	2009-12-01 02:02:14.372266700 -0600
@@ -10,8 +10,8 @@ import java.io.IOException;
 class OrgLookupTest {
     public static void main(String[] args) {
 	try {
-	    LookupService orgl = new LookupService("/usr/local/share/GeoIP/GeoIPOrg.dat");
-	    LookupService ispl = new LookupService("/usr/local/share/GeoIP/GeoIPISP.dat");
+	    LookupService orgl = new LookupService("/usr/share/GeoIP/GeoIPOrg.dat");
+	    LookupService ispl = new LookupService("/usr/share/GeoIP/GeoIPISP.dat");
 	    System.out.println("Organization: " + orgl.getOrg(args[0]) +
 			       "\tISP: " + ispl.getOrg(args[0]));
 	    orgl.close();
--- origsrc/GeoIPJava-1.2.3/RegionLookupTest.java	2009-05-11 04:02:44.000000000 -0500
+++ src/GeoIPJava-1.2.3/RegionLookupTest.java	2009-12-01 02:02:19.738676100 -0600
@@ -8,7 +8,7 @@ import java.io.IOException;
 class RegionLookupTest {
     public static void main(String[] args) {
         try {
-            LookupService cl = new LookupService("/usr/local/share/GeoIP/GeoIPRegion.dat");
+            LookupService cl = new LookupService("/usr/share/GeoIP/GeoIPRegion.dat");
             Region l = cl.getRegion(args[0]);
             System.out.println("Country Code: " + l.countryCode);
             System.out.println("Country Name: " + l.countryName);
