--- origsrc/GeoIPJava-1.2.1/CountryLookupTest.java	2006-02-28 23:29:52.000000000 -0600
+++ src/GeoIPJava-1.2.1/CountryLookupTest.java	2007-10-22 12:36:24.742743400 -0500
@@ -15,7 +15,7 @@
 	    // String dir = System.getProperty("user.dir"); 
 
 	    // Uncomment for Linux
-	    String dir = "/usr/local/share/GeoIP";
+	    String dir = "/usr/share/GeoIP";
 
 	    String dbfile = dir + sep + "GeoIP.dat"; 
 	    // You should only call LookupService once, especially if you use
